friends/src/Components/FriendList.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import {connect} from 'react-redux';
+
+import Friend from "./Friend";
+import {getFriends} from "../Actions"
+
+
+class FriendList extends React.Component{
+    state={
+        friends:[],
+        loadingFriends:[],
+    }
+
+    componentDidMout(){
+        this.props.getFriends();
+    }
+
```

Comment:
Line 14 - typo for the life cycle method `"componentDidMout"` (should be `componentDidMount()` - missing an `'n'` character).